### ISSUE TITLE
Is RecordID always unique?

### ISSUE THREAD

**Issue by micnguyen** [Alterplay/APAddressBook#169]
I have noticed that when I delete a contact from the address book and then adding another contact, that it doesn't overwrite the removed ID and instead adds a new one. Is the `recordID` for an `APContact` always unique? I'm trying to sync up a custom domain model to an `APContact` sych that each APContact represents a 1:1 custom model and am using `recordID`. Is the record ID similar to that of `Identifier` in https://developer.apple.com/documentation/contacts/cncontact/identifier?

**Comment by belkevich** [comment]
I don't think that `recordID` matches `Identifier`. `APAddressBook` is a wrapper on deprecated `AddressBook.framework` not `Contacts.framework`.

**Comment by belkevich** [comment]
But if you try, please, tell us your results
